.github/workflows/build-release.yml

```diff
@@ -1,39 +0,0 @@
-name: Build and publish app release
```

Comment:
I think both should keep existing, that way we can still make a release if we want to.

Reply:
I tried too keep the original job and let the new one just create the release, thinking it would trigger a build, but it did not.